.markdownlint.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
What is the purpose of this file? Does it need to be submitted?

Reply:
It allows us to configure lint rules for markdown files. More information is available here: https://github.com/DavidAnson/markdownlint

Reply:
Prettier works on markdown too (including formatting code inside code blocks).

Reply:
Another question is, what if we just use Prettier for markdown instead of markdownlint? Or can they complement each other?

Reply:
@trusktr Prettier and markdownlint serve different purposes but complement each other: 

* Prettier is designed to format code. By doing so, it enforces standards.
* markdownlint is designed to enforce standards. It can automatically fix some (but not all) errors which ends up formatting code.

Here's a simple example:

```markdown
## Title 1


- Item
- Item


#### Title 2


* Item
* Item
```

Prettier will remove the extra line breaks between content:

```markdown
## Title

- Item
- Item

#### Title 2

* Item
* Item
```

Markdownlint will provide the following error message:

* Document should start with an `h1` heading
* Heading levels should only increment/decrement one level at a time
* List style types should be consistent (don't mix `-` and `*` in the same document)

Regardless, I'm going to remove `.markdownlint.json` from the PR and handle markdown linting in a separate PR. :)